Global Date/Time/Number Formats: Add API Descriptions (#8697)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/GlobalConfig/dateFormat.md

---
id: GlobalConfig.dateFormat
type: LocalizationFormat | Record
default: undefined
---
---
##### shortDescription
Specifies the default date format for all DevExtreme components in the application.

---

> This property can accept a [predefined format string](/Documentation/ApiReference/Common_Types/#Format), a [custom format string](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) or a [format function](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Function).

Reviewer: ```suggestion
This property can accept a [predefined format string](/Documentation/ApiReference/Common_Types/#Format), [custom format string](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String), or [format function](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Function).
```

> Assign a string value to this property to use a [predefined format](/Documentation/ApiReference/Common_Types/#Format). Specify a [custom format string](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) or a [format function](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Function) to use a custom format.

Reviewer 2: The flow is not ideal... You start with "a string value" and associate that with a "predefined format". Then you list "a custom format string" as an alternative... But it's also a string value, right? 

I want to simplify the description to a list of three options:

```suggestion
This property can accept a [predefined format string](/Documentation/ApiReference/Common_Types/#Format), a [custom format string](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) or a [format function](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Function).
```

---

##### jQuery

    <!-- tab: index.js -->
    DevExpress.config({
        dateFormat: 'yyyy-MM-dd',
    });

##### Angular

    <!-- tab: app.component.ts -->
    import config from "devextreme/core/config";

    config({
        dateFormat: 'yyyy-MM-dd',
    });

##### Vue

    <!-- tab: App.vue -->
    import config from "devextreme/core/config";

    config({
        dateFormat: 'yyyy-MM-dd',
    });

##### React

    <!-- tab: App.tsx -->
    import config from "devextreme/core/config";

    config({
        dateFormat: 'yyyy-MM-dd',
    });

---

> You can configure default formats for specific locales. To do this, assign a key-value pair object to this property. Use locale identifiers as keys. Use the `default` key to specify a format for all other locales:

Reviewer: ```suggestion
You can configure default formats for specific locales. Assign an object with key-value pairs to this property. Use locale identifiers as keys. Use the `default` key to specify a format for all other locales:
```

> You can configure default formats for specific locales. To do this, assign a key-value pair object to this property. Use locale identifiers as keys to define formats for specific locales. Use the `default` key to specify a format for all other locales:

Reviewer 2: Just removing half a sentence that I think is redundant. 

```suggestion
You can configure default formats for specific locales. To do this, assign a key-value pair object to this property. Use locale identifiers as keys. Use the `default` key to specify a format for all other locales:
```
